unit_tests/test_utils__operator__multitenant_common.py

```diff
@@ -197,7 +197,7 @@ def _multitenant_class_with_unique_options(self, klass):
 
         # Process tenant-2 specific options
         self.assertEqual(
-            tenants[1].params.get("stress_cmd_r"), "fake__stress_cmd_r__tenant2__single_str_in_list")
+            tenants[1].params.get("stress_cmd_r"), ["fake__stress_cmd_r__tenant2__single_str_in_list"])
```

Comment:
It must not be changed, why the `stress_cmd_r` option is expected to behave differently here?

Reply:
yes, if fixed it already